src/app/ad-authoring/generate-story-html.ts

```diff
@@ -0,0 +1,584 @@
+import {generateAdAmpHtml, generateAmpHtmlParams} from './generate-ad-html';
+
+export function generateStoryAmpHtml(ampHtml: generateAmpHtmlParams) {
+  const adAmpHtml = generateAdAmpHtml({
+    callToActionStr: ampHtml.callToActionStr,
+    landingUrl: ampHtml.landingUrl,
+    landingType: ampHtml.landingType,
+    assetSrc: ampHtml.assetSrc,
+    assetFile: ampHtml.assetFile,
+  });
+  const storyAmpHtml = `<!doctype html>
```

Comment:
Optional: It's ok to return this value directly instead of assigning to a constant first.